contracts/contracts/vault/VaultStorage.sol

```diff
@@ -77,7 +77,6 @@ contract VaultStorage is Initializable, Governable {
     // Strategies approved for use by the Vault
     struct Strategy {
         bool isSupported;
-        uint256 _deprecated; // Deprecated storage slot
```

Comment:
This will affect any future changes to the struct (since any new prop added might already have values set for existing strategies). Uniswap strategy already does that. It's probably low impact but I'd prefer not to do it

Reply:
That's why we need to do this now. The Uniswap V3 Strategy will add another bool after this which means we have two bools using three storage slots.

Reply:
I get your point. But how do we make sure that we don't accidentally use the old value in slot1 (set before deprecation) when we add more vars to this? Right now, having this named `_depreacted` acts as a reminder for us

Reply:
All the old `slot1` values are set to zero which is the default value.

Here's the OETH Vault
- https://evm.storage/eth/17540186/0x39254033945aa2e4809cc2977e7087bee48bd7ab/strategies.0x1827f9ea98e0bf96550b2fc20f7233277fcd7e63#map
- https://evm.storage/eth/17540186/0x39254033945aa2e4809cc2977e7087bee48bd7ab/strategies.0x3ff8654d633d4ea0fae24c52aec73b4a20d0d0e5#map
- https://evm.storage/eth/17540186/0x39254033945aa2e4809cc2977e7087bee48bd7ab/strategies.0xc1fc9e5ec3058921ea5025d703cbe31764756319#map

Here's the OUSD Vault `strategies` mapping which you can drill into each mapping to check the `slot1` values
https://evm.storage/eth/17540105/0xe75d77b1865ae93c7eaa3040b038d7aa7bc02f70/strategies#map

Reply:
Oh, it makes more sense to remove it then